pages/about.html

```diff
@@ -1,12 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/common.css">
+  <link rel="stylesheet" href="/style.css">
+  <title>About</title>
 </head>
+<header>
+  <h1>About Me</h1>
+</header>
+
+<nav>
+  <ul>
+    <li><a href="index.html">Home</a></li>
+    <li><a href="about.html">About</a></li>
+    <li><a href="portfolio.html">Portfolio</a></li>
+  </ul>
+</nav>
+
 <body>
```

Comment:
The body tag should contain everything here but the head tag. Basically
```html
<html>
<head><!-- all the head tags, like title and stylesheets --></head>
<body><!-- all the content, including header and footer --></body>
</html>
```

I think you mentioned that moving the body tag around was something you had been experimenting with, and it just ended up here? They would be better served with the main tag here.

Running a validator can help us detect when we misuse tags.